src/main/java/seedu/itlogger/ItLogger.java

```diff
@@ -65,6 +65,11 @@ public static void main(String[] args) {
 
             case VIEW: // Shi Jie
                 // todo -> view ONE Defect with INDEX NUMBER
+                try {
+                    System.out.println(issueList.getDefect(parseIndex(input)).toString());
+                } catch (Exception e) {
```

Comment:
@ZhengShijieNUS @pigoliver @JanuariusJang 
Should we place the entire switch case statement in one TRY block & all catch statement after the TRY block for cleaner and more readable code design?